phac-nml/irida-next
# Workflow selection UI

## What does this PR do and why?
_Describe in detail what your merge request does and why._

Layout for the selection of workflows.

- 1 workflow is currently hardcoded into the HTML for demonstration purposes and will be updated

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

![image](https://github.com/phac-nml/irida-next/assets/11295750/fb47f4e6-02ed-49a9-9c52-eb871653bb91)

- White is the un-hovered state.
- Green is the hovered state.

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

Go to a project samples page, and click on the rocket icon to open the workflow launch modal.  There you will see the layout.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
